enterprise/models.py

```diff
@@ -1244,6 +1245,24 @@ def _complete_user_enrollment():
                         discount_percentage=enrollment.discount_percentage,
                         sales_force_id=enrollment.sales_force_id,
                     )
+
+                    # If the pending enrollment was using a license, generate a license enrollment alongside the
+                    # enterprise enrollment.
+                    if enrollment.license_uuid:
+                        source = EnterpriseEnrollmentSource.get_source(EnterpriseEnrollmentSource.ENROLLMENT_URL)
```

Comment:
I'm not 100% sure what this source is, if anyone could enlighten me as to why we use it in the first bulk enrollment, I'd appreciate it 😄

Reply:
I think it is just to track who created this enrollment, sometimes it's via django admin sometimes via an api call etc.. not 100% clear how it's used